fava/static/javascript/format.ts

```diff
@@ -69,3 +69,10 @@ export let currentTimeFilterDateFormat = timeFilterDateFormat.month;
 interval.subscribe(intervalValue => {
   currentTimeFilterDateFormat = timeFilterDateFormat[intervalValue];
 });
+
+export function dateToString(date: Date): string {
```

Comment:
This can use a formatter from d3-time-format, like dateFormat.day (but using `timeFormat` instead of `utcFormat` I guess)

Reply:
You're right, `timeFormat` gives the same result as my implementation.